processing/src/main/java/io/druid/query/aggregation/JavaScriptAggregatorFactory.java

```diff
@@ -172,7 +172,7 @@ public AggregatorFactory getMergingFactory(AggregatorFactory other) throws Aggre
               @Override
               public AggregatorFactory apply(String input)
               {
-                return new JavaScriptAggregatorFactory(input, fieldNames, fnAggregate, fnReset, fnCombine, config);
+                return new JavaScriptAggregatorFactory(input, Lists.newArrayList(input), fnCombine, fnReset, fnCombine, config);
```

Comment:
Apart from javascript, implementation of getRequiredColumn for other aggregatorFactories seems to work fine without converting to combiningFactory